extension.js

```diff
@@ -83,48 +106,85 @@ async function activate() {
 	});
 }
 
-async function setColor(color) {
+/**
+ * Checks if the active editor is in a Salesforcxe project
+ * @returns {boolean} true if the active editor is in a Salesforce project
+ */
+async function checkIfInSfProject() {
+	const sfdxConfigFiles = await vscode.workspace.findFiles(
+		OLD_PATH || NEW_PATH,
+		null,
+		1
+	);
+
+	if (sfdxConfigFiles.length > 0) {
+		return true;
+	}
+	return false;
+}
+
+/**
+ *
+ * @param {string} backgroundColor - hex color code for background color
+ * @param {string} [foregroundColor] - hex color code for foreground color (Optional)
+ */
+async function setColor(backgroundColor, foregroundColor) {
 	const config = vscode.workspace.getConfiguration();
+	const settingsScope =
+		config.get('sf-colorg.target.settingsScope') || 'workspace';
+	const target =
+		settingsScope === 'workspace'
+			? vscode.ConfigurationTarget.Workspace
+			: vscode.ConfigurationTarget.Global;
```

Comment:
@jlawrence124 suggestion: what do you think about extracting this piece of code into a reusable function, as we're using it later also inside the initialCleanup function?

Reply:
done!

https://github.com/victorgz/vscode-salesforce-colorg/pull/4/commits/32d208c6fea2c6ef6785f57e7f70e57318c32852